### ISSUE TITLE
Margin and padding responsive in UIKit 3 ?

### ISSUE THREAD

**Issue by rilril99** [uikit/uikit#2908]
I think responsive margin and padding will be nice in next version . Hope to see a new fix !

**Comment by DanielDarrenJones** [comment]
+1 for this, would really like to be able to change the margin and padding based on the screen size!

**Comment by FistMeNaruto** [comment]
Also having `padding-left`, `padding-top` and others, just like `margin-left` and `margin-right` would be useful. As of now, you have to use `uk-padding uk-padding-remove-vertical uk-padding-remove-right` to add left padding.

**Comment by garethwarry** [comment]
Agreed, this would be very nice. Something like `uk-margin-remove uk-margin-small@m` would add a small margin for medium and larger screens, while not adding a margin for smaller screens.

**Comment by Uranbold** [comment]
Padding-left
Padding-right
Padding-bottom
Padding-top

Padding-small-left
Padding-small-right
Padding-small-bottom
Padding-small-top

Padding-large-left
Padding-large-right
Padding-large-bottom
Padding-large-top

:)

**Comment by erniomaldo** [comment]
Hi there, you can add custom css to do that, something like this: 
`@media (min-width: 960px) {
  .uk-padding-remove-bottom\@m {
    padding-bottom: 0px !important;
  }
}`
Edit the css to match to match what you're looking for.

**Comment by PaulFelgentreff** [comment]
If you're working with scss 

- create  a new mixin: padding-mixin.scss with the code (1)
- add the file to your site.scss (2)
- compile

**(1)**
`@mixin hook-transition-misc(){

    /* Remove
     ========================================================================== */

     /* Phone landscape and bigger */
    @media (min-width: $breakpoint-small) {

        .uk-padding-remove-left\@s { padding-left: 0 !important; }
        .uk-padding-remove-right\@s { padding-right: 0 !important; }

    }

    /* Tablet landscape and bigger */
    @media (min-width: $breakpoint-medium) {

        .uk-padding-remove-left\@m { padding-left: 0 !important; }
        .uk-padding-remove-right\@m { padding-right: 0 !important; }

    }

    /* Desktop and bigger */
    @media (min-width: $breakpoint-large) {

        .uk-padding-remove-left\@l { padding-left: 0 !important; }
        .uk-padding-remove-right\@l { padding-right: 0 !important; }

    }

    /* Large screen and bigger */
    @media (min-width: $breakpoint-xlarge) {

        .uk-padding-remove-left\@xl { padding-left: 0 !important; }
        .uk-padding-remove-right\@xl { padding-right: 0 !important; }

    }
}`

**(2)**
`// site.scss

// 1. Your custom variables and variable overwrites.

// 2. Import default variables and available mixins.

// 3. Your custom mixin overwrites.
@import "eyefox/padding-mixins.scss";`

**Comment by yoyhosoft** [comment]
Any update?

**Comment by Messa1** [comment]
would be nice to see that in uikit
